### PR TITLE
AO3-6155 Changed wording for History preference

### PR DESCRIPTION
**Pull request by hocestcorpus**
## Issue

https://otwarchive.atlassian.net/browse/AO3-6155

## Purpose

Reword text relating to the History preference. 

## Testing Instructions

Refer to Jira.

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*

(If you have a Jira account, please include the same name in the "Full name"
field on your Jira profile, so we can assign you the issues you're working on.)

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

(If you do not fill in this section, we will use your GitHub account name and
they/them pronouns to avoid making assumptions about your gender.)

### CODE REVIEW

File: public/help/misc-preferences.html [hunk 1]
```diff
@@ -2,8 +2,8 @@ <h4>Miscellaneous Preferences</h4>
 
 <dl id="help">
   
-  <dt>Turn on Viewing History</dt>
-  <dd>When enabled, viewing history keeps a log of every work you view on the Archive while logged in. You can delete individual works from your History or clear the whole History. If you enable this option and subsequently disable it, works viewed while the option was enabled will still be saved (but you will have to enable the Viewing History again to see them).</dd>
+  <dt>Turn on History</dt>
+  <dd>When enabled, History keeps a log of every work you view on the Archive while logged in. You can delete individual works from your History or clear the whole History. If you enable this option and subsequently disable it, works viewed while the option was enabled will still be saved (but you will have to enable History again to see them).</dd>
```
Comment (sarken) [inline public/help/misc-preferences.html:6]: As long as we're here, would you mind changing it from

* "every work you view" to "every work you access"
* "works viewed while the option" to "works accessed while the option"

**Review by sarken — CHANGES_REQUESTED** [review]
Thank you for the pull request! I'll update the text to adhere to our usual PR template, but if you submit any more PRs in the future, we'd appreciate it if you could stick to the template. 

The changes themselves look good, but I'm afraid there are some tests that need updating as well before this is ready to merge. I also noticed two instances of "view" in the help text that I forgot to mention in the issue, so if you're willing to fix those as well, that would be great.

Thank you again!

File: public/help/misc-preferences.html (hunk 1 above)
Comment (hocestcorpus) [reply public/help/misc-preferences.html]: Thank you for taking the time to leave an explanatory comment! I pushed another commit with the changes you detailed. One check failed (failure to reach server), but the same check passed when I ran the test suite on my fork of the code, so I'm not sure if that reflects an issue in the code or a situational network timeout or something of that sort.

**Review by sarken — APPROVED** [review]
This looks good -- thanks for making the additional changes! 

I'm rerunning the tests one more time, but I agree that the failure isn't related to these changes (I'm getting it on a different branch myself), so I'm going to mark this ready to merge.

**Pull request merged**